### PR TITLE
pre work

### PR DESCRIPTION
**Pull request by aleksandra-charatonik-red**

### CODE REVIEW

File: test/GeneratorsTests.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        auto numberOfValues = 10;
+        collection.push_back(GeneratorType::FIBONACCI);
+
+        WHEN("sumNumbersFromGenerators is called")
+        {
+            auto result = sumNumbersFromGenerators(collection, numberOfValues);
+
+            THEN("It should return same value as sumGeneratedNumbers")
+            {
+                CHECK(result == sumGeneratedNumbers(std::move(std::make_unique<FibonacciNumberGenerator>()), numberOfValues));
```
Comment (ziobron) [inline test/GeneratorsTests.cpp]: `std::move()` is redundant here. The code will work in the same way without it. The same for other places.

File: test/GeneratorsTests.cpp [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            }
+        }
+    }
+
+    GIVEN("One Fibonacci generator, one prime generator and numberOfValues = 10")
+    {
+        std::vector<GeneratorType> collection = {};
+        auto numberOfValues = 10;
+        collection.push_back(GeneratorType::FIBONACCI);
+        collection.push_back(GeneratorType::PRIME);
```
Comment (ziobron) [inline test/GeneratorsTests.cpp]: You can also initialize the collection immediately:
```cpp
std::vector<GeneratorType> collection = {GeneratorType::FIBONACCI, GeneratorType::PRIME};
```

**Review by ziobron — APPROVED** [review]
Looks good :)

File: test/GeneratorsTests.cpp (hunk 1 above)
Comment (ziobron) [reply test/GeneratorsTests.cpp]: The compiler also complains that it prevents optimisation: https://travis-ci.com/coders-school/pre_work/jobs/260930593

Comment (aleksandra-charatonik-red) [reply test/GeneratorsTests.cpp]: True :) corrected

**Pull request closed without merging**